Dockerfile

```diff
@@ -52,6 +59,7 @@ COPY requirements/local-inference.txt requirements-local-inference.txt
 
 RUN python3.8 -m pip install pip==${PIP_VERSION} \
   && pip install --no-cache -r requirements-base.txt \
+  && pip install --no-cache -r requirements-test.txt \
```

Comment:
did adding this add much time to install and size to the image? probably easiest for now, but wondering as we try to slim this down what kind of impact this has.

Reply:
Not a ton, but as Austin mentioned below maybe there's a better way to bring in test dependencies! Going to play around with that some more.